manuscript/060_Statement_First_Programming.md

```diff
@@ -135,39 +134,41 @@ var configuration = new XmlConfiguration(customFixture);
+“Test-After" ends up as “Test-Never" 
 ------------------------------------
 
-I will ask this question again: ever had to write a�requirement or design document for something that you already implemented? Was it fun? Was it valuable? Was it creative? No, I�do not think so. The same is with our executable specification. After we write the code, we have little motivation to specify what is already written -- some of the pieces of code �we can just see are correct", other pieces �we already saw working" when we copied our code over to our deployment machine and ran few sanity checks... The design is ready... Specification? Maybe next time...
+I will ask this question again: did you ever have to write a requirements or design document for something that you already implemented? Was it fun? Was it valuable? Was it creative? No, I do not think so. The same holds for our executable specification. After we've written the code, we have little motivation to specify what we wrote -- some of the pieces of code “we can just see are correct", other pieces “we already saw working" when we copied the code over to our development machine and ran a few sanity checks... The design is ready... Specification? Maybe next time...
 
-Another reason might be time pressure. Let's be honest -- we are all in a�hurry, we are all under pressure and when this pressure is too high, it triggers heroic behaviors in us, especially when there is a�risk of not making it with the sprint commitment. Such heroic behavior usually goes by the following rules: drop all the �baggage", stop learning and experimenting, revert to all of the old �safe" behaviors and �save what we can!". If Specification is written at the end, it is often sacrificed on the altar of making it with the commitment, since the code is already written, �and it will be tested anyway by real tests" (box tests, smoke tests, sanity tests etc.). It is quite the contrary when starting with a�Statement, where the Statement evaluating to false is **a reason** to write any code. Thus, if we want to write code, Specification become irremovable part of your development. By the way, I�bet in big corporations no one sane ever thinks they can abandon checking in the code to source control, at the same time treating Specification as �an optional addition".
+Another reason might be time pressure. When pressure becomes too high, it seems to trigger heroic behaviors in us. Then all of a sudden we drop all the “baggage", stop learning and experimenting, revert to all of the old “safe" behaviors and “save what we can!". If the Specification is written at the end, it is sacrificed, since the code already has been written, “and it will be tested anyway by real tests" like box tests, smoke tests and sanity tests. However, when starting with a Statement it is quite the opposite. Here a Statement evaluating to false is **a reason** to write any code. Thus, creating the Specification is an indispensable part of writing code.
```

Comment:
How about:

Another reason might be time pressure and fear of deadlines. Under these circumstances, the teams I worked in often showed a willingness to sacrifice something "for the greater good". Many times this "greater good" was seen in meeting the deadline. Rather sacrifice something users won't notice than sacrifice something they will might be the reasoning. It then becomes attractive to skip writing a Specification, especially when it is created after the production code has been written. Besides, "the code will be tested anyway by *real tests*" like black-box tests, smoke tests and sanity tests.

Starting with a Statement brings a different point of view, where a false Statement is **a reason** to write any code. Thus, creating the Specification is an indispensable part of writing code.